repair/repair.cc

```diff
@@ -29,7 +29,6 @@
 #include "streaming/stream_reason.hh"
 #include "gms/inet_address.hh"
```

Comment:
Every revert has to explain why it is reverting.

Reply:
Done, added this information to every revert commit.